protractor.conf.js

```diff
@@ -11,7 +11,7 @@ require('dotenv').config({ path: process.env.ENV_FILE });
 
 const SmartRunner = require('protractor-smartrunner');
 const projectRoot = path.resolve(__dirname);
-const downloadFolder = `${projectRoot}/e2e-downloads`;
+const downloadFolder = path.join(__dirname, '/e2e-downloads');
```

Comment:
this change does not make any sense unless you remove '/' (non-windows) separator from the folder name. The 'path.jon' is automatically adding separators. But why we need this change anyway?

Reply:
Regarding the first question - I've added the forward slash separator to be consistent with ADF protractor.conf.js e.g.
![image](https://user-images.githubusercontent.com/26852253/89526423-a2f70d00-d7df-11ea-8ec5-f0523c7fd440.png)
Double forward slashes in paths also work on Windows

Regarding the second question -- I've raised this ticket as we have 3 people in our team running e2e locally on Windows machines and the suggested fix should make e2e run on any OS without failures.